internal/util/state/state.go

```diff
@@ -26,8 +26,9 @@ import (
 
 // State represents FerretDB process state.
 type State struct {
-	UUID      string `json:"uuid"`
-	Telemetry *bool  `json:"telemetry,omitempty"` // nil for undecided
+	UUID           string        `json:"uuid"`
+	Telemetry      *bool         `json:"telemetry,omitempty"` // nil for undecided
+	UndecidedDelay time.Duration `json:"undecided_delay,omitempty"`
```

Comment:
I added it to the state as this info is needed in the warning text. Let me know if you have a better idea.

Reply:
Let’s remove it from the text instead

Reply:
fixed